lib/tools/adb-commands.js

```diff
@@ -674,4 +674,37 @@ methods.getDeviceProductLocale = async function () {
   return await this.getDeviceProperty("ro.product.locale");
 };
 
+methods.getModel = async function () {
+  let stdout = await this.shell('getprop');
+  let model = new RegExp(/ro.product.model]:([^\r?\n]+)*/g).exec(stdout);
+  if (model && model.length >= 2) {
```

Comment:
I think it would be more convenient to rewrite the condition this way:

```javascript
if (model && model.length >= 2) {
  return model[1].trim().replace(/[\[\]]/g, '');
}
return null;
```

Same to the methods below